client/src/elm/Translate.elm

```diff
@@ -21162,6 +21183,12 @@ translationSet trans =
         TuberculosisReasonMedicationsNotDistributed reason ->
             translationSet <| TuberculosisReasonNotProvidedToday reason
 
+        TuberculosisSuspect ->
+            { english = "Tuberculosis Suspect"
+            , kinyarwanda = Nothing
+            , kirundi = Just "Nta Gupima/gusuzuma"
```

Comment:
This is not the equivalent of the English string. This means: `No Testing/Assessment`

Reply:
Thanks @AronNovak. Looks like a result of copy/paste and forgetting to replace values with Nothing.